packages/analytics/client/src/analytics_client/analytics_client.ts

```diff
@@ -133,12 +133,9 @@ export class AnalyticsClient implements IAnalyticsClient {
       properties: eventData as unknown as Record<string, unknown>,
     };
 
-    // debug-logging before checking the opt-in status to help during development
-    if (this.initContext.isDev) {
-      this.initContext.logger.debug<EventDebugLogMeta>(`Report event "${eventType}"`, {
-        ebt_event: event,
-      });
-    }
+    this.initContext.logger.debug<EventDebugLogMeta>(`Report event "${eventType}"`, {
+      ebt_event: event,
+    });
```

Comment:
No longer needs this `isDev` check now that the default level is `info` and is configurable.

Reply:
then again https://xkcd.com/1172 :trollface: